### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by afizsavage**
In this PR:
- We added an email column to the owner's table
- We added a large amount of data to the owners and visits tables
- We created indexes for owners and visits tables to improve response time for certain queries. 

Below are the screenshot for the results of our queries before and after optimization.

**Query 1 before**
1_before
**Query 1 after**
1_after

**Query 2 before**
2_before
**Query 2 after**
2_after

**Query 3 before**
3_before
**Query 3 after
3_after
**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -43,4 +43,10 @@ ALTER TABLE animals ALTER COLUMN id ADD GENERATED BY DEFAULT AS IDENTITY;
...
 ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD species_id INT REFERENCES species(id);
-ALTER TABLE animals ADD owner_id INT REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD owner_id INT REFERENCES owners(id);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX ON visits (animal_id);
+CREATE INDEX vets_idx ON visits (vet_id);
```
Comment (AshakaE) [inline schema.sql:51]: - [ ] Kindly remove this index and add it again, also take screenshots of the `explain analyze` output once more

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @direakanbi  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by afizsavage** [comment]
May I ask which one of the indexes to remove?

File: schema.sql (hunk 1 above)
Comment (afizsavage) [reply schema.sql:51]: Should I do that for for the vet_id index only?

Comment (afizsavage) [reply schema.sql:51]: Screenshot from 2022-03-08 19-39-02

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @afizsavage 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**